mantidproject/mantid
# Indirect - Diffraction Reduction - Correctly implement Use Manual Grouping

In the Indirect Diffraction Reduction interface, the 'Use Individual Grouping' option does not work as intended and currently has no function in the interface. This option should behave similarly to the 'Groups' mode of the the Detector Grouping option in ISISEnergyTransfer.

This PR ensures the Use Individual Grouping option (now the Use Manual Grouping option, in order to avoid confusion with Individual Grouping mode in ISISEnergyTransfer)  works as intended.

**To test:**
1. Ensure that you have 'Use Data Archive' enabled.
2. Navigate to Interfaces > Indirect > Diffraction.
3. Enter 26176 into the Sample field.
4. Check the checkbox labelled 'Use Manual Grouping' and ensure the spinner labelled 'Number of Groups' is enabled.
5. Select any number of groups > 0 and note the number of groups selected.
6. Run the Algorithm.
7. Open the output workspace and ensure the number of spectra is equal to the number of selected groups (spectra has been summed into N groups).



Fixes #20477 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
